tests/tests.js

```diff
@@ -81,7 +71,7 @@ describe('Discord Tests', function() {
          * Test that the hook returns a 200 when posted to
          */
         it('Hook returns a 200 OK when posted to', function(done) {
-            sendPost('1', function(error, response, body) {
+            sendPost(testUtils.recordedFixturesDir + '1', function(error, response, body) {
```

Comment:
`sendHookPayload`? The thing that tripped me up was that the `/hook` url was hard-coded in the function.